VMRecovery/ResourceManager/New-AzureRMRescueVM.ps1

```diff
@@ -64,7 +64,7 @@
 .EXAMPLE
     Example for managed disk VM:
 
-    $scriptResult =  .\New-AzureRMRescueVM.ps1 -resourceGroupName recoveryVMRg -VmName recovmtestmg -subscriptionId xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx -UserName "sujasd" -Password "XPa55w0rrd12345" -prefix "rescuex1" 
+    $scriptResult =  .\New-AzureRMRescueVM.ps1 -resourceGroupName sujasrg -VmName sujunmanagedP -subscriptionId xx-xx-xx -UserName "sujasd" -Password "XPa55w0rrd12345" -prefix "rescuex0011" 

```

Comment:
This is a bit odd to change the usage to be more specific.  Is it intentional?  I notice the SubId "mask" is no longer accurate, perhaps either change it back to the write mask or put an actual (generated) example GUID in there.